web/buttons.jsp

```diff
@@ -198,6 +198,68 @@
                     .children("a").attr("onclick","deleteHotkey("+position+");");
                 },"html");           
             });
+            
+            $("#updateChars").click(async function(){
+                var listItems = $("#sortable1 li");
+                var chars = [];
+                listItems.each(function(i, li) {
+                    chars[i] = parseInt($(li).children(".shrink")[0].value);
+                });
+                var allCharData = {"projectID":projectID, "chars":chars}
+                await fetch("updateSpecialCharacters", {
+                    method: "POST",
+                    mode: "cors",
+                    headers: {
+                        'Content-Type': 'application/json;charset=utf-8'
+                    },
+                    body: JSON.stringify(allCharData)
+                })
+                .then(response => {
+                    if(response.ok){
+                        $("#buttonForm").submit();
+                    }
+                })         
+            });
```

Comment:
a catch with an alert that the page didnt save correctly might be handy to have here.